app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -333,8 +332,6 @@ def authenticator_view_data
 
   def generic_data
     {
-      personal_key_unavailable: personal_key_unavailable?,
```

Comment:
`personal_key_unavailable?` is now unused, right? could we remove it?

Reply:
Good catch, removed